app/actions/LendingRequestActions.ts

```diff
@@ -0,0 +1,54 @@
+import { LendingRequest } from 'app/actions/ActionTypes';
+import callAPI from 'app/actions/callAPI';
+import { lendingRequestSchema } from 'app/reducers';
+import type { EntityId } from '@reduxjs/toolkit';
+import type { LendingRequest as LendingRequestModel } from 'app/store/models/LendingRequest';
+
+export function fetchAllLendingRequests() {
+  return callAPI<LendingRequestModel>({
```

Comment:
```suggestion
  return callAPI<LendingRequestModel[]>({
```
This seems incorrect based on the schema